distribution/test/itests/test-itests-dependencies-app/pom.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>itests</artifactId>
+        <groupId>org.codice.alliance.test.itests</groupId>
+        <version>0.2-SNAPSHOT</version>
+    </parent>
+    <artifactId>test-itests-dependencies-app</artifactId>
+    <name>Alliance :: Integration :: Test Dependencies App</name>
```

Comment:
I propose naming this package `Alliance :: Test :: Integration :: Dependencies App`